public_html/wp-content/mu-plugins/blocks/package.json

```diff
@@ -21,6 +21,9 @@
 		"rememo": "3.0.0"
 	},
 	"devDependencies": {
+		"@wordpress/babel-preset-default": "4.4.0",
```

Comment:
You don't need to import this Babel preset anymore if you use `@wordpress/scripts` v3.4.0. This was a bug which should be resolved now.

I opened https://github.com/WordPress/gutenberg-examples/pull/84 to explore all of this myself and I identified a couple of things which we can simplify on Gutenberg side to make it more seamless. I will work on it in the following weeks.

Reply:
Thanks for pointing that out – I started out by updating the `@wordpress/*` packages but was running into some `eslint` issues. Tried again just now and it's fine - I think part of it might've been that I wanted to add `eslint-plugin-jest` to our config, but then had trouble replicating it to debug.

Reply:
> I think part of it might've been that I wanted to add `eslint-plugin-jest` to our config, but then had trouble replicating it to debug.

I will handle it as part of https://github.com/WordPress/gutenberg/issues/17016. It might take a few weeks until it gets released though. As soon as you want to override config, you have to bring defaults and this might mean more dependencies. That's why it's so important to do the recommended configs correctly. We will refine `@wordpress/scripts` to make it seamless for real :)

Reply:
Yep, that's why I've been working to remove as much of our overrides as we can 😁

Reply:
(but getting the lint errors in the test file was why I started down the eslint-jest path, I ended up just adding a comment to the file in the end, like you did with https://github.com/WordPress/gutenberg-examples/pull/84 )